lib/nfs_common.pm

```diff
@@ -247,8 +247,13 @@ sub start_service {
     # Back on the console
     clear_console;
 
+    # Duo to poo#114875, the below journalctl command may take several time to
+    # get return code in serial terminal on low performance setups. so enhance
+    # the logic not to check the return code and add some sleep here to make
+    # the test more stable.
+    background_script_run "( journalctl -fu nfs-server -o short-precise > /dev/$serialdev )";
+
```

Comment:
I'm starting to think that this is meant to only get some logs, and nothing is actually depending on this call... (i.e, there's no wait_serial  that is looking for any string that might come out of the journal)... Perhaps we can simply remove it, and use `save_upload_logs` instead?

Reply:
I can't agree with you any more! ：）
Delete this line will impact the function IMO, only one concern is `short-precise` may give more detail logs there.
@pdostal, what is your sugestion?

Reply:
https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&test=qam-minima_base_RFAN

Reply:
@foursixnine @pdostal 
https://openqa.suse.de/tests/9291896#